src/components/pages/CreateNote.js

```diff
@@ -0,0 +1,95 @@
+import PropTypes from 'prop-types'
+import React, { useState } from 'react'
+import { useHistory, useParams } from 'react-router-dom'
+import { v4 as uuidv4 } from 'uuid'
+import ButtonPrimary from '../form/Button'
+import InputText from '../form/InputText'
+import TextEditor from '../form/TextEditor'
+import styled from 'styled-components/macro'
+
+CreateNote.propTypes = {
+  setTrips: PropTypes.func,
+  trips: PropTypes.array,
+}
+
+export default function CreateNote({ setTrips, trips }) {
+  const uniqueId = uuidv4()
```

Comment:
You call `uuidv4()` on every render, but it is only needed once in `handleSubmit`.